### ISSUE TITLE
Wand of Fortune R Nintendo Switch Script doesn't find text

### ISSUE THREAD

**Issue by poetxBLUE** [0xDC00/scripts#428]
The script for the Nintendo Switch version of Wand of Fortune R needs a fix. The Agent doesn't find any text to translate when I play, I even tried to just skip a bunch of text and see if it works further into the game, but nothing. Thanks for the support!

I used the Version 0.1.4.20240428 (nightly) of Agent and yuzu Early Access 4176

**Comment by poetxBLUE** [comment]
i used

**Comment by Mansive** [comment]
The script works for me. Is your game version 1.0.0?

**Comment by poetxBLUE** [comment]
Yes, the game version adds up and Agent also doesn't show any error, it just shows no text, same as the overlay, it just won't advance any text, even though I advance in the game. Also sorry for closing and reopening, I missclicked there.
